src/lib.ts

```diff
@@ -413,17 +413,20 @@ export function getBanishedMonsters(): Map<Item | Skill, Monster> {
     if (foe === undefined || banisher === undefined) break;
     // toItem doesn"t error if the item doesn"t exist, so we have to use that.
     const banisherItem = toItem(banisher);
-    const banisherObject = [
-      Item.get("none"),
-      Item.get(`training scroll:  Snokebomb`),
-      Item.get(`tomayohawk-style reflex hammer`),
-      null,
-    ].includes(banisherItem)
-      ? banisher.toLowerCase() === "saber force"
-        ? Skill.get(`7311`)
-        : Skill.get(banisher)
-      : banisherItem;
-    result.set(banisherObject, Monster.get(foe));
+    if (banisher.toLowerCase() === "saber force") {
+      result.set(Skill.get(`7311`), Monster.get(foe));
+    } else if (
+      [
+        Item.get("none"),
+        Item.get(`training scroll:  Snokebomb`),
+        Item.get(`tomayohawk-style reflex hammer`),
+        null,
+      ].includes(banisherItem)
+    ) {
+      result.set(Skill.get(banisher), Monster.get(foe));
```

Comment:
Do we want to check that this isn't $skill[none]? You and I know that it won't happen for snokebomb or reflex hammer, but it could happen for null or $item[none]